src/main/java/Main.java

```diff
@@ -0,0 +1,55 @@
+import figure.AbstractFigure;
+import figure.Circle;
+import figure.Square;
+import figure.Trapezoid;
+import figure.Triangle;
+
+import static figure.Colors.getRandomColors;
+
+public class Main {
```

Comment:
```suggestion
public class FigureGenerator {
```
it would be better that you created a class for this particular purpose
then in `Main` class you can create `new FigureGenerator()` and call methods to generate and draw the figures